Tasks/google-play-release/task.loc.json

```diff
@@ -14,7 +14,7 @@
   ],
   "version": {
     "Major": "3",
-    "Minor": "168",
+    "Minor": "171",
```

Comment:
Could you please bump it to 172 in task.json/task.loc.json files? Since current sprint is 172